h2o-r/tests/testdir_algos/generic/runit_generic_model_mojo_stackedensemble.R

```diff
@@ -0,0 +1,55 @@
+source("generic_model_test_common.R")
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../../scripts/h2o-r-test-setup.R")
+
+test.model.generic.stackedensemble <- function() {
+    train <- h2o.uploadFile(locate("smalldata/testng/higgs_train_5k.csv"))
+    y <- "response"
+    x <- setdiff(names(train), y)
+    train[, y] <- as.factor(train[, y])
+    nfolds <- 5
+    
+    gbm <- h2o.gbm(
+        x = x,
+        y = y,
+        training_frame = train,
+        distribution = "bernoulli",
+        ntrees = 10,
+        nfolds = nfolds,
+        fold_assignment = "Modulo",
+        keep_cross_validation_predictions = TRUE,
+        seed = 1
+    )
+    
+    randomforest <- h2o.randomForest(
+        x = x,
+        y = y,
+        training_frame = train,
+        ntrees = 5,
+        nfolds = nfolds,
+        fold_assignment = "Modulo",
+        keep_cross_validation_predictions = TRUE,
+        seed = 1
+    )
+    stackedensemble <- h2o.stackedEnsemble(
+        x = x,
+        y = y,
+        training_frame = train,
+        base_models = list(gbm@model_id, randomforest@model_id)
+    )
+    print(stackedensemble)
+    mojo_original_name <- h2o.download_mojo(model = stackedensemble, path = tempdir())
+    mojo_original_path <- paste0(tempdir(),"/",mojo_original_name)
+    
+    generic_model <- h2o.genericModel(mojo_original_path)
+    print(generic_model)
+    
+    generic_model_preds  <- h2o.predict(generic_model, train)
+    expect_equal(length(generic_model_preds), 3)
+    expect_equal(h2o.nrow(generic_model_preds), 5000)
```

Comment:
do we check the predictions anywhere?

Reply:
JUnit: https://github.com/h2oai/h2o-3/pull/4416/files#diff-87f75a070de23cd6342c51d04c308a6bR905

Reply:
Also, check here please: https://github.com/h2oai/h2o-3/pull/4416/files#diff-87f75a070de23cd6342c51d04c308a6bR908